src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -107,7 +115,9 @@ def __call__(self, poller):
         from msrest.exceptions import ClientException
         logger.info("Starting long running operation '%s'", self.start_msg)
         correlation_message = ''
+        self.progress_controller.begin()
         while not poller.done():
+            self.progress_controller.add(message='Running')
```

Comment:
We already have a start/end message as part of the long running operation instance. Should we use the start message instead of a hard-coded Running message?